docs/platforms.md

```diff
@@ -44,10 +44,10 @@ The following platforms are supported, with a different set of features availabl
 * kubevirt
   - Attributes
   - SSH Keys
-* openstack
+* openstack (from config-drive, then possibly fallback to the Nova metadata service)
```

Comment:
Why dont we document in a similar way so to follow the format fount here => https://github.com/coreos/afterburn/pull/931/commits/6a06a02666d88450650c15136970f66be5322d7c#diff-7d38dbc6beeb2c1c6db5d85d6d58d87b81a09aee1f02ac76be60caf734a075aa